adler32.c

```
@@ -8,34 +8,53 @@
+
+#  else
+//Fallback to preferred function
+uint32_t Z_EXPORT zng_adler32(uint32_t adler, const unsigned char *buf, size_t len) {
+    if (buf == NULL)
+        return ADLER32_INITIAL_VALUE;
+    return FUNCTABLE_CALL(adler32)(adler, buf, len);
+}
+#  endif
+
+// Deprecated function
+uint32_t Z_EXPORT zng_adler32_z(uint32_t adler, const unsigned char *buf, size_t len) {
```

Cppcheck (2.20.0)

[style] The function 'zng_adler32_z' is never used.

(unusedFunction)